en/features-guide/index.rst

```diff
@@ -0,0 +1,70 @@
+QGEP / TEKSI wastewater module
+===========
+
+TEKSI wastewater module (projet name: QGEP) is a selection of tools and a database implementation that allows you to:
+
+* manage and map your seewer network data with all its components in compliance with Swiss standards
+* indicate the characteristics of networks objects in the form of attributes such as diameter, material, depth, damages, construction year, etc.
+* produce plans and extract statistics from the database such as network values, total length of pipes, identification of future interventions, etc.
+* export geodata in compliance with Swiss standards
+
+Main features
+------------
+
+**Network objects**
+
+Creation, information and management of network elements according to the VSA-DSS model:
+
+* Wastewater structures
+* Special structures
+* Primary collectors
+* Secondary collectors (ex : private connection)
+* Catchments
+* GEP mesures
+* Maintenances
+* Network inspection (VSA-DSS model extension VSA-KEK)
+
+**Viewing and editing**
+
+A preconfigured QGIS project including the symbology, the forms and the relation between the objets is provided.
+
+The editing can be done directly using the QGIS natives tools and the QGEP complementary toolbox (ex : inversion of the direction of the pipes and its related altitudes).
+
+**Files**
+
+Documents, pictures and videos can be linked to the network elements.
+
+**Networks following tools**
+
+Network topology management:
+
+* Topology creation between pipes, structures and catchments
+* Upstream and downstream network tracking tools (water drop)
+* Length profiles display
+
+**Mapping and statistics production**
```

Comment:
I would not mention this. Users can do this if they know QGIS. But in the defaut project there is no preconfigured layout and no really good symbology (for other themes as Werkplan). My opinion.

Reply:
For the serial plotting we should provide at least a sample print composer (as we do for QWAT)

https://github.com/QGEP/QGEP/issues/707

For statistics extractions I'm open to suggestions for real life examples, we could provide some specific statistics views.

Reply:
@varrieta should we consider that actual users are QGIS / GIS well known users or should we mention functionnalities only if they are well defined on the modules?

Open to suggestions

Reply:
I think that anyone using QGEP will have to learn  at one point to plot a map. But it is through that for now there is no preconfigured layout, we can erase that and let map production ?